code_schemes/s01mag07.json

```diff
@@ -3,6 +3,110 @@
   "Name": "S01MAG07",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "DisplayText": "youth are the future",
+      "VisibleInCoda": true,
+      "NumericValue": 0,
+      "CodeID": "code-227b079b",
+      "CodeType": "Normal",
+      "StringValue": "youth_are_the_future"
+    },
+    {
+      "DisplayText": "backbone_of_the_society",
+      "VisibleInCoda": true,
+      "NumericValue": 1,
+      "CodeID": "code-2cd02361",
+      "CodeType": "Normal",
+      "StringValue": "backbone_of_the_society"
+    },
+    {
+      "DisplayText": "youth_are_the_success_of_the_community",
+      "VisibleInCoda": true,
+      "NumericValue": 2,
+      "CodeID": "code-28853c86",
+      "CodeType": "Normal",
+      "StringValue": "youth_are_the_success_of_the_community"
+    },
+    {
+      "DisplayText": "part of the society",
+      "VisibleInCoda": true,
+      "NumericValue": 3,
+      "CodeID": "code-42463ad8",
+      "CodeType": "Normal",
+      "StringValue": "part_of_the_society"
+    },
+    {
+      "DisplayText": "majority of the population",
+      "VisibleInCoda": true,
+      "NumericValue": 4,
+      "CodeID": "code-05197dcb",
+      "CodeType": "Normal",
+      "StringValue": "majority_of_the_population"
+    },
+    {
+      "DisplayText": "youth are discriminated",
+      "VisibleInCoda": true,
+      "NumericValue": 5,
+      "CodeID": "code-f5bc5b1d",
+      "CodeType": "Normal",
+      "StringValue": "youth_are_discriminated"
+    },
+    {
+      "DisplayText": "decisions are made by elders and leaders",
+      "VisibleInCoda": true,
+      "NumericValue": 6,
+      "CodeID": "code-6b729a58",
+      "CodeType": "Normal",
+      "StringValue": "decisions_are_made_by_elders_and_leaders"
+    },
+    {
+      "DisplayText": "clannism as the major obstacle",
```

Comment:
as or is?

Reply:
Thank you updated to -> changed to clannism is a major obstacle re- RDA